lib/logstash/inputs/syslog.rb

```diff
@@ -79,12 +83,12 @@ def register
     require "thread_safe"
     @grok_filter = LogStash::Filters::Grok.new(
       "overwrite" => "message",
-      "match" => { "message" => "<%{POSINT:priority}>%{SYSLOGLINE}" },
+      "match" => { "message" => @grok_pattern },
       "tag_on_failure" => ["_grokparsefailure_sysloginput"],
     )
 
     @date_filter = LogStash::Filters::Date.new(
-      "match" => [ "timestamp", "MMM  d HH:mm:ss", "MMM dd HH:mm:ss", "ISO8601"],
+      "match" => [ "timestamp", "MMM dd HH:mm:ss", "MMM  d HH:mm:ss", "ISO8601"],
```

Comment:
I added a teeny, tiny optimization here, as 2/3 of the month has double-digits, and first match wins. 😄